command/ssh/certificate.go

```diff
@@ -254,6 +261,28 @@ func certificateAction(ctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
+
+		// All host identity certs need a URI SAN to work with our ssh API.
+		if isHost {
+			var u = uuid.Nil
+			if hostID == "" {
+				u, err = deriveMachineID()
+				if err != nil {
+					return err
+				}
+			} else {
+				u, err = uuid.Parse(hostID)
+				if err != nil {
+					return err
```

Comment:
Wrap this using the `pkg/errors` package, we are always wrapping errors in the CLI.

Reply:
Or probably better to use an error like `errs.InvalidFlagValue(...)`